edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/device/DeviceMessageHandler.cs

```diff
@@ -234,6 +235,86 @@ async Task HandleTwinOperationException(string correlationId, Exception e)
             }
         }
 
+        #region IDeviceProxy
+
+        public Task SendC2DMessageAsync(IMessage message) => this.underlyingProxy.SendC2DMessageAsync(message);
```

Comment:
I'm trying to figure out why this diff is so big and what changed. Is this new code?

Reply:
No, I just moved product code to be above the Events class (it got moved around during the cleanup sometiem back). No changes here other than Metrics class.